addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -51,6 +54,32 @@ var QuestionFormWidget = publicWidget.Widget.extend({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     *
+     * @param commentInput
+     * @private
+     */
+    _onCommentChanged: function (event) {
+        var input = event.currentTarget;
+        var commentIcon = $(input).closest('.o_wslides_js_quiz_answer').find('.o_wslides_js_quiz_comment_answer');
+        if (input.value.trim() !== '') {
+            commentIcon.addClass('filled');
```

Comment:
You can use text-primary class or any other bootstrap standard coloring class instead of custom css. That way themes are correctly applied.